.travis.yml

```diff
@@ -277,6 +277,9 @@ jobs:
           echo $v dmesg ======================== ;
           docker exec -it druid-$v sh -c 'dmesg | tail -3' ;
           done
+        - for v in ~/shared/tasklogs/*.log ; do
+          echo $v logtail ======================== ; tail -100 $v ;
```

Comment:
This could cause travis failure since it would fail once the number of output rows reaches to the limit.

Reply:
I'll revert this for now